### ISSUE TITLE
withe pop up osticket V1.11

### ISSUE THREAD

**Issue by LeopoldoZD** [osTicket/osTicket#4753]
![problem](https://user-images.githubusercontent.com/16768191/53209440-1747eb80-3600-11e9-8e1e-cdf7b9016edd.png)
![information](https://user-images.githubusercontent.com/16768191/53209449-1dd66300-3600-11e9-9c4a-04ad4e5f1ec0.png)
Hello again
how can I solve this problem that when creating a user, it is osticket v1.11

**Comment by JediKev** [comment]
@LeopoldoZD

I cannot reproduce this using the latest v1.11. Please take a look at NGINX error logs as well as php error logs for any related errors.

Cheers.

**Comment by JediKev** [comment]
@LeopoldoZD

Also, NGINX is not a supported web server so this may be the cause as well. There are extra settings you must put in the the NGINX configuration file to make these work. Have you taken a look at this yet?
https://www.nginx.com/resources/wiki/start/topics/recipes/osticket/

Cheers.

**Comment by tricoos** [comment]
I have the same problem. The result returned is an HTTP 400 Bad Request with just "URL not supported":
The error log does not help as there is no NGINX error. The error comes from osTicket which returns a 400, probably because the request is not rewritten correctly:
`GET /scp/ajax.php/content/log/1 HTTP/1.1" 400`

ANd this happens with the recipe at https://www.nginx.com/resources/wiki/start/topics/recipes/osticket/

Anyway, NGINX should be supported because it is (with roughly 33%) the second (!) most utilized web server: https://trends.builtwith.com/web-server. It's not a niche product.

**Comment by JediKev** [comment]
@tricoos 

> The error log does not help as there is no NGINX error. The error comes from osTicket which returns a 400, probably because the request is not rewritten correctly

I'm not able to reproduce this with my instance of NGINX, PHP v7.2, and v1.11 upgraded so it must be something with either your instance of NGINX, something with the database, or something with the code. In any case it should be throwing errors in your PHP, NGINX, browser console, or osTicket error logs as this is a hard crash, and typically with crashes there are errors reported in the logs. I see that your AJAX request is pointing to "content/log/" so it appears as though it's trying to log something to osTicket. Go to **Admin Panel > Dashboard > System Logs** to see if there are any related logs/errors.

> Anyway, NGINX should be supported because it is (with roughly 33%) the second (!) most utilized web server

I've said this in many other threads and I'll say it again here...NGINX is not a supported webserver at this time because it takes extra configurations, etc. to get it working with osTicket properly, and even then there are no guarantees. With Apache/IIS there is nothing you really need to do to get it working out of the box. We are going to support most, if not all, popular webservers in the near future. Stay tuned.

Cheers.

**Comment by tricoos** [comment]
Reason: PATH_INFO is empty
I fixed this by replacing "$path_info" with "$path_info2" in https://www.nginx.com/resources/wiki/start/topics/recipes/osticket/ 
And this is how my PHP block looks like - I actually did NOT use this block from the URL because NGINX has its own snippet - however this set $path_info to an empty string:

```
location ~ \.php$ {
        include snippets/fastcgi-php.conf;
        fastcgi_param  PATH_INFO        $path_info2;
        # With php-fpm (or other unix sockets):
        fastcgi_pass unix:/var/run/php/php7.2-fpm.sock;
        # With php-cgi (or other tcp sockets):
        # fastcgi_pass 127.0.0.1:9000;
}
```

**Comment by JediKev** [comment]
@tricoos 

See, an issue with NGINX...that's why we don't support it yet 😉

Cheers.

**Comment by tricoos** [comment]
@JediKev

> NGINX is not a supported webserver at this time because it takes extra configurations, etc. to get it working with osTicket properly, and even then there are no guarantees. 

Yes, but it's working with Apache just because you created the .htaccess files in the first place - it didn't happen magically ;)

> With Apache/IIS there is nothing you really need to do to get it working out of the box. We are going to support most, if not all, popular webservers in the near future. Stay tuned.

Ok, so now with the support for # 1 (Apache) and # 3 (IIS) hopefully # 2 (NGINX) will be supported soon.
